ui/components/UploadImport.js

```diff
@@ -101,14 +113,13 @@ const UploadImport = ({ handleUpload, handleUrlUpload, configuration, isApplicat
   return (
     <>
       <label htmlFor="url-upload-button">
-
-        <Button aria-label="URL-Upload" data-cy="import-button" variant="contained"
-          color="primary" className={classes.button}
-          size="large" onClick={handleOpen}>
-          <LinkIcon style={{ paddingRight : ".35rem" }} />
-          Import {configuration}
-        </Button>
-
+        <Tooltip title="Upload File">
+          <div className={classes.customToolbarIcons}>
+            <IconButton aria-label="Upload" component="span" onClick={handleOpen}>
+              <PublishIcon />
```

Comment:
@Harshit-Dandriyal This will change the import button globally, which might not go with the current UX. Could you provide an ss of how that looks in Meshery UI in configurations?

Reply:
Yes, You are correct.